src/frontend/components/UI/DLCList/DLC/index.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  faCancel,
+  faCloudArrowDown,
+  faSpinner,
+  faTrash
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import ContextProvider from 'frontend/state/ContextProvider'
+import { hasProgress } from 'frontend/hooks/hasProgress'
+
+type Props = {
+  dlc: DLCInfo
+  runner: Runner
+  mainAppInfo: GameInfo
+  onClose: () => void
+}
+
+const DLC = ({ dlc, runner, mainAppInfo, onClose }: Props) => {
+  const { title, app_name } = dlc
+  const { libraryStatus, showDialogModal } = useContext(ContextProvider)
+  const { t } = useTranslation('gamepage')
+  const [showUninstallModal, setShowUninstallModal] = useState(false)
+  const [dlcInfo, setDlcInfo] = useState<GameInfo | null>(null)
+  const [dlcSize, setDlcSize] = useState<number>(0)
+  const [refreshing, setRefreshing] = useState(true)
+  const [progress] = hasProgress(app_name)
+
+  const isInstalled = dlcInfo?.is_installed
+
+  useEffect(() => {
+    const checkInstalled = async () => {
+      const info = await getGameInfo(app_name, runner)
+      if (!info) {
+        return
+      }
+      setDlcInfo(info)
+    }
+    checkInstalled()
+  }, [dlc, runner])
+
+  useEffect(() => {
+    setRefreshing(true)
+    const getDlcSize = async () => {
+      if (!mainAppInfo.install.platform) {
+        return
+      }
+      const info = await getInstallInfo(
+        app_name,
+        runner,
+        mainAppInfo.install.platform
+      )
+      if (!info) {
+        return
+      }
+      setDlcSize(info.manifest.download_size)
+      setRefreshing(false)
+    }
+    getDlcSize()
+  }, [dlc, runner])
+
+  const currentApp = libraryStatus.find((app) => app.appName === app_name)
+  const isInstalling = currentApp?.status === 'installing'
+  const showInstallButton = !isInstalling && !refreshing
+
+  function mainAction() {
+    if (isInstalled) {
+      setShowUninstallModal(true)
+    } else {
+      const {
+        install: { platform, install_path }
+      } = mainAppInfo
+
+      if (!dlcInfo || !platform || !install_path) {
+        return
+      }
+      onClose()
+      install({
+        isInstalling,
+        previousProgress: null,
+        progress,
+        showDialogModal,
+        t,
+        installPath: install_path,
+        gameInfo: dlcInfo,
+        platformToInstall: platform
+      })
+    }
+  }
+
+  return (
+    <>
+      {showUninstallModal && (
+        <UninstallModal
+          appName={app_name}
+          runner={runner}
+          onClose={() => setShowUninstallModal(false)}
+          isDlc
+        />
+      )}
+      <div className="dlcItem">
+        <span className="title">{title}</span>
+        {refreshing ? '...' : <span className="size">{size(dlcSize)}</span>}
+        {showInstallButton && (
+          <span className="action" onClick={() => mainAction()}>
+            <FontAwesomeIcon
+              icon={isInstalled ? faTrash : faCloudArrowDown}
```

Comment:
I think we should use the same download icon we use in the library or the wine manager for consistency. Here it looks like this:

![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/c3a461b7-4959-400e-b6bb-b2c5f110db63)

but in the library it's:

![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/da4b3229-e0dd-47ab-a41b-4e8b30b73c8f)

Reply:
<img width="475" alt="image" src="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/26871415/75c18d0b-6034-44ab-bb55-bf3804f8bef6">

Reply:
fixes